### PR TITLE
Update import project step

### PR DESCRIPTION
**Pull request by bencefr**
This PR updates the import project step for SEGGER Embedded Studio v1.5+. The text has been copied over from the toolchain manager app.

### CODE REVIEW

File: resources/data/ses_config-0.1.2.json [hunk 1]
```diff
@@ -43,15 +43,20 @@
...
-                        "- **CMakeLists.txt** - the location of the ``CMakeLists.txt`` project file of the sample that you want to work with (for example, the full path to ``ncs/nrf/applications/asset_tracker/CMakeLists.txt``)",
-                        "- **Board Directory** - the location of the board description for which to build the project (for example, the full path to ``ncs/zephyr/boards/arm/nrf9160_pca10090``)",
-                        "- **Board Name** - the board name for which to build the project (for example, ``nrf9160_pca10090ns``)",
-                        "- **Build Directory** - the folder in which to run the build (for example, the full path to ``ncs/nrf/applications/asset_tracker/build``)",
-                        "- **Delete Existing CMakeCache.txt** - select this option to ensure that you are not building with an outdated build cache",
+                        "- **nRF Connect SDK Release** - Select the nRF Connect SDK version that you want to work with.",
+                        "- **nRF Connect Toolchain Version** - Select NONE and make sure that your SES environment is configured correctly.",
```
Comment (ru-fu) [inline resources/data/ses_config-0.1.2.json:51]: Can you have a link to the docs here? I think I would be confused if I'm only being told to check if my environment is configured correctly, but not how it should be configured.
In the docs, we have "Select NONE and make sure that your SES environment is configured correctly (see Setting up the SES environment)." with link to https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/gs_installing.html#setting-up-ses

File: resources/data/ses_config-0.1.2.json [hunk 2]
```diff
@@ -43,15 +43,20 @@
...
+                        "- **Projects** - Select the project that you want to work with.",
+                        "    Select any of the checkboxes to add samples and applications from the respective area to the drop-down list. To add custom projects to the drop-down list, click  and select the folder that contains the projects that you want to add.",
```
Comment (ru-fu) [inline resources/data/ses_config-0.1.2.json:53]: Double space between "click" and "and" - not sure if that shows in the output.
The same in line 55.

File: resources/data/ses_config-0.1.2.json (hunk 1 above)
Comment (bencefr) [reply resources/data/ses_config-0.1.2.json:51]: Yes, however that is part of the previous step. So could this be even simpler?

Comment (ru-fu) [reply resources/data/ses_config-0.1.2.json:51]: Sure, if we know this is configured correctly, it's sufficient to just say "Select NONE."

Comment (ru-fu) [reply resources/data/ses_config-0.1.2.json:51]: It's probably also the only option in the drop-down in this case, isn't it?

**Review by ru-fu — APPROVED** [review]
Looks good!

**Pull request merged**